src/core/namespace.h

```diff
@@ -234,6 +234,9 @@ ProcSubset proc_subset_from_string(const char *s) _pure_;
 const char* private_bpf_to_string(PrivateBPF i) _const_;
 PrivateBPF private_bpf_from_string(const char *s) _pure_;
 
+void bpf_delegate_to_string(uint64_t u, char *s);
```

Comment:
How about something like the following?
```c
#define BPF_DELEGATE_STRING_MAX (STRLEN("0x") + sizeof(uint64_t) * 2 + 1)
char* bpf_delegate_to_string(uint64_t u, char buf[static BPF_DELEGATE_STRING_MAX]);
#define BPF_DELEGATE_TO_STRING(u) bpf_delegate_to_string(u, (char[BPF_DELEGATE_STRING_MAX]){})
```
Then, you can use e.g.
```c
r = serialize_item(f, "exec-context-bpf-delegate-commands", BPF_DELEGATE_TO_STRING(c->bpf_delegate_commands));
``

Reply:
is the assert still needed with buf declared as array?